SingularityService/src/main/java/com/hubspot/singularity/resources/WebhookResource.java

```diff
@@ -50,6 +52,24 @@ public WebhookResource(WebhookManager webhookManager, SingularityAuthorizationHe
     return webhookManager.getActiveWebhooks();
   }
 
+  @GET
+  @Path("/summary")
+  @ApiOperation("Retrieve a summary of each active webhook")
+  public List<SingularityWebhookSummary> getActiveWebhookSummaries() {
+    authorizationHelper.checkAdminAuthorization(user);
+    List<SingularityWebhookSummary> webhookSummaries = new ArrayList<SingularityWebhookSummary>();
```

Comment:
Two things here. First, let's move this into the WebhookManager class, so we can manage building everything there. Second, there are a few methods available in the WebhookManager class that should allow us to do this more efficiently (batch up the zookeeper calls). Using something like the `getAsyncChildren` calls could help here.